runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -157,7 +174,7 @@ void KernelGenerator::visit(const ir::operation::Conv2D &node)
 
   auto fn = std::make_unique<ops::ConvolutionLayer>();
 
-  fn->configure(ifm_alloc, ker_alloc, bias_alloc, padding_type, padding.left, padding.right,
+  fn->configure(ifm_alloc, ker_alloc, bias_alloc, parm_padding, padding.left, padding.right,
```

Comment:
Why do you change `padding_type` to `parm_padding`?

Reply:
It need padding information in calculatePadding so I changed it. But I think it can be modified more neatly. I will change it.